src/commands/basic/init.js

```diff
@@ -66,6 +66,92 @@ const showInstructions = () => {
   makeInitialCommit();
 };
 
+/**
+ * Fetch the user reqired test runner
+ *
+ * @param {String} template
+ * @returns {Promise<Void>}
+ */
+const fetchTestRunner = async template => {
+  const { testRunner } = await inquirer.prompt([
+    {
+      name: 'testRunner',
+      type: 'list',
+      message: 'Please select your test framework for your project',
+      choices: ['none', 'ava', 'jest', 'mocha'],
+    },
+  ]);
+  if (testRunner === 'none') {
+    return;
+  }
+  if (testRunner === 'ava' && template === 'nuxt') {
+    return;
+  }
```

Comment:
It seems `Nuxt.js` works pretty much with `Ava`. Can you clarify about this early return?